pkg/template/master_template.go

```diff
@@ -279,7 +279,7 @@ systemd:
       ExecStartPre=/usr/bin/mkdir -p /opt/bin/
       ExecStartPre=/usr/bin/docker pull $IMAGE
       ExecStartPre=-/usr/bin/docker rm $CONTAINER_NAME
-      ExecStartPre=-/usr/bin/docker create --name $CONTAINER_NAME $IMAGE
+      ExecStartPre=-/usr/bin/docker create --name $CONTAINER_NAME $IMAGE /kubectl
```

Comment:
`hyperkube` doesn't define a command so we have to specify one to be able to create the container. We're not running the container so it doesn't matter otherwise.